dashboard/test/models/pd/workshop_test.rb

```diff
@@ -351,7 +351,17 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     # Make a FiT workshop that's ended and has attendance;
     # these are the conditions under which we'd normally send a survey.
     workshop = create :pd_ended_workshop, subject: SUBJECT_FIT
-    create(:pd_workshop_participant, workshop: workshop, enrolled: true)
+    create(:pd_workshop_participant, workshop: workshop, enrolled: true, attended: true)
+
+    # Ensure no exit surveys are sent
+    Pd::Enrollment.any_instance.expects(:send_exit_survey).never
+    workshop.send_exit_surveys
+  end
+
+  test 'send_exit_surveys sends no surveys for Facilitator workshops' do
```

Comment:
Weird diff - this is the new test.  I also pulled one line from the test above, since it wasn't relevant to the asserted idea.